### PR TITLE
change default message variable name from SLACK_MESSAGE to DEFAULT_SL…

### PR DESCRIPTION
**Pull request by kaiomagalhaes**
…ACK_MESSAGE, add async option to send the notification with tests

### CODE REVIEW

**Comment by kaiomagalhaes** [comment]
Regarding to #5 and #2

**Comment by kaiomagalhaes** [comment]
@seathony @halissonvit @codelitt

**Comment by codelitt** [comment]
LGTM :+1:

File: lib/slacked/slack_post.rb [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
+
   class << self
+    def post message = ENV[SLACK_DEFAULT_MESSAGE_KEY]
+      return false if message.nil? || message.empty?
+      notifier = slack_notifier
+      notifier.ping message, SLACK_CONFIG
+    end
+
+    def post_async message
+      Thread.start { post(message) }
```
Comment (Lockyy) [inline lib/slacked/slack_post.rb]: Do we need to close the database connection that is created by the Thread here? I've not used threads much but remember reading about each thread opening a DB connection that needs closing.

Comment (Lockyy) [inline lib/slacked/slack_post.rb]: Found a link here: https://www.agileplannerapp.com/blog/building-agile-planner/rails-background-jobs-in-threads

**Pull request merged**